code/modules/projectiles/guns/magnetic/bore.dm

```diff
@@ -98,7 +98,16 @@
 			capacitor = null
 			update_icon()
 			return
-
+			if(!manipulator)
+				to_chat(user, "<span class='warning'>\The [src] has no manipulator installed.</span>")
+				return
+			manipulator.forceMove(get_turf(src))
+			user.put_in_hands(manipulator)
+			user.visible_message("<span class='notice'>\The [user] unscrews \the [manipulator] from, \the [src].</span>")
```

Comment:
Should probs remove the comma after "from"

Reply:
Done and done!